core/domain/exp_domain.py

```diff
@@ -232,7 +232,7 @@ def __init__(
                 made in this model, which should give sufficient information to
                 reconstruct the commit. Each dict always contains the following
                 key:
-                    - cmd: str. Unique command.
+                    - cmd: (str). Unique command.
```

Comment:
Could you please keep this as it was? We typically don't put parens around typeinfo.

Reply:
Done